modules/services/getmail.nix

```
@@ -0,0 +1,61 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.getmail;
+
+  accounts = filter (a: a.getmail.enable)
+    (attrValues config.accounts.email.accounts);
+
+  renderConfigFilepath = a: ".getmail/getmail${a.name}";
+  quoted = concatMapStrings (x: " --rcfile \${HOME}/${x}") (map renderConfigFilepath accounts);
```

Could simplify slightly:
```suggestion
  quoted = concatMapStringsSep " " (a: "--rcfile ${renderConfigFilepath a}")  accounts;
```

---

Ah yep. Cheers for this!